R/build-reference-index.R

```diff
@@ -163,7 +159,7 @@ default_reference_index <- function(pkg = ".") {
 
 check_missing_topics <- function(rows, pkg, error_call = caller_env()) {
   # Cross-reference complete list of topics vs. topics found in index page
-  all_topics <- rows %>% purrr::map("names") %>% unlist(use.names = FALSE)
+  all_topics <- purrr::list_c(purrr::map(rows, "names"))
```

Comment:
I was starting to wonder how different that was from vctr, I see it's the same :relieved: